supervisord.conf.tmpl

```diff
@@ -34,6 +34,12 @@ stopwaitsecs = 60
 # If the process didn't kill it's children after 60 seconds, it's unlikely to
 # ever reap them, so kill them all
 killasgroup = true
+stdout_logfile = %(here)s/var/log/celery.log
+stdout_logfile_maxbytes = %(sup_log_size_celery_out)s
+stdout_logfile_backups = %(sup_log_backups_celery_out)s
+stderr_logfile = %(here)s/var/log/celery.err
```

Comment:
Can't change `%(here)s/var/log` into a variable as well so you don't repeat this path, also gives flexibility on where the logs can go. So, 2 variables to tune, but defaults to /var/log in assembl directory.